README.md

```diff
@@ -194,7 +194,8 @@ Builtin validators:
 |Variable Type|Description|
 |-------------|-----------|
 |`checkbox`|Ensure the value is 0 or 1.|
-|`select`|Ensure the options list is not empty.|
```

Comment:
Why was `select` removed?

Reply:
This error is thrown by the parser, not the validator:
https://github.com/StylishThemes/parse-usercss/blob/ff91ec2740647f1c2671967fec8b223c68396df3/lib/parse.js#L121-L125